servers.json

```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "id" : "Eclipsepvp" ,
```

Comment:
The `id` field should be lowercase. You are also missing your server logo in the `logos` directory